### PR TITLE
Add Documentation to PB Group 2

### PR DESCRIPTION
**Pull request by megantrimble**
#### Checklist:

- [ ] **SPECS** Please cover your changes with specs
- [x] **CHANGELOG** Please add an entry on `[Unreleased]` section to every functionality `ADDED`/`CHANGED`/`DEPRECATED`/`REMOVED`/`FIXED`

### CODE REVIEW

**Comment by megantrimble** [comment]
Form_1
Form_2
Form_3

**Comment by megantrimble** [comment]
Radio_1
Radio_2
Radio_3

**Comment by megantrimble** [comment]
Select_1
Select_2
Select_3

**Comment by megantrimble** [comment]
Select_Card_1
Select_Card_2
Select_Card_3
Select_Card_4

**Comment by megantrimble** [comment]
Text_Input_1
Text_Input_2
Text_Input_3

**Comment by megantrimble** [comment]
Textarea_1
Textarea_2
Textarea_3

**Comment by megantrimble** [comment]
Typeahead_1

**Comment by megantrimble** [comment]
Toggle_1
Toggle_2

**Comment by megantrimble** [comment]
Highlight_1
Highlight_2

**Comment by megantrimble** [comment]
Hashtag_1

**Comment by megantrimble** [comment]
Icon_1
Icon_2
Icon_3
Icon_4

**Comment by megantrimble** [comment]
Icon_Circle_1

File: app/pb_kits/playbook/pb_form/docs/_description.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Form displays containers with labels and fields for a user to fill out / input information. Typically used on cards or white backgrounds.
```
Comment (jasoncypret) [inline app/pb_kits/playbook/pb_form/docs/_description.md]: @thestephenmarshall This kit is a bit technical. Can you help Megan provide a bit more detail about what this kit is? I think a bit more explanation about how this kit is used for forms in rails perhaps

**Pull request merged**